### DISCUSSION TITLE
Salesforce Custom Object Mode instructions issue

### DISCUSSION THREAD

**Discussion by davidcsi2002** [amazon-connect/amazon-connect-salesforce-scv#115] [General]
Hi there,
We are setting up AWS using Salesforce custom object mode following this instruction:
https://github.com/amazon-connect/amazon-connect-salesforce-scv/blob/master/Solutions/VMX2-VoicemailExpress/Docs/vmx_prerequistes.md

We found the instruction might need to be updated.
Step 4 in section "Create a custom field for the voicemail callback number" and "Create a custom field for the contact Attributes".
It says select object Case. In our test, it should be selecting the custom object (which is Voicemail__c in this case).

And I also would suggest add a section regarding setting the Name field to Text instead of Auto number.

Thanks,
David

**Comment by dougjaso** [comment]
Yeah, that's a good catch. I'll change it. Thanks
